ts/components/ui/__test__/ActivityIndicator.test.tsx

```diff
@@ -0,0 +1,9 @@
+import TestRenderer from "react-test-renderer";
+import React from "react";
+import ActivityIndicator from "../ActivityIndicator";
+
+describe("ActivityIndicator", () => {
+  it("Freeze 'ActivityIndicator' state", () => {
```

Comment:
I'd rather keep the "standard" phrasing: "should match the snapshot with default props" or something alike.
The reason is to make it easier to backtrace failures on the CI and enable some degree of automation/reporting based on keywords.

Reply:
done within https://github.com/pagopa/io-app/pull/3229/commits/49c6fec0c3d3309c000aed322ca19b69c0ce209d